### PR TITLE
F #18 webauftrit

### PR DESCRIPTION
**Pull request by Laszlo32**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -109,6 +114,7 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+
```
Comment (a-kraschitzer) [inline pom.xml]: remove space

File: src/main/resources/templates/fragments/header.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<html
```
Comment (a-kraschitzer) [inline src/main/resources/templates/fragments/header.html]: there should be different headers for
* not signed in
* signed in as customer
* signed in as staff 

as they all have different locations they can access and different pages they can open

File: src/main/resources/templates/fragments/header.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    <link rel="stylesheet" type="text/css" href="menu.css"/>
+</head>
+<body>
+<center>
+    <h1>Software Endearing Seminar</h1><br/>
+    <h2>Hotel management site</h2><br/><br/>
+    <div id="navcontainer" th:fragment="menu">
+        <ul>
+            <li><a href="login" th:href="@{/login}">Login</a></li>
+            <li><a href="#">Search Room</a></li>
```
Comment (a-kraschitzer) [inline src/main/resources/templates/fragments/header.html]: why is there no link for the button? "staff/roomDirectory"

File: src/main/resources/templates/staff/index.html [hunk 4]
```diff
@@ -16,9 +17,12 @@
 </div>
 <h1>Hello Spring Security</h1>
 <p>Welcome to the STAFF space!</p>
+<div th:insert="fragments/header :: menu"></div>
+<!--
 <ul>
```
Comment (a-kraschitzer) [inline src/main/resources/templates/staff/index.html]: remove comments when they are note needed anymore

File: src/main/resources/templates/customer/index.html [hunk 5]
```diff
@@ -15,9 +16,12 @@
     </div>
 </div>
 <h1>Hello Spring Security</h1>
+<div th:insert="fragments/header :: menu"></div>
 <p>Welcome to the CUSTOMER space!</p>
+<!--
 <ul>
     <li>Go to the <a href="/index" th:href="@{/index}">unsecured pages</a></li>
 </ul>
+-->
```
Comment (a-kraschitzer) [inline src/main/resources/templates/customer/index.html]: remove code, if it is replaced with other code

File: src/main/resources/templates/fragments/header.html [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+            <li sec:authorize="isAuthenticated()"><a href="/customer/index" th:href="@{/customer/index}">customerEntity pages</a></li>
+            <li sec:authorize="hasRole('STAFF')"><a href="/staff/index" th:href="@{/staff/index}">staff pages</a></li>
+            <!--<li><a href="#"></a></li>-->
```
Comment (a-kraschitzer) [inline src/main/resources/templates/fragments/header.html]: remove comment

File: src/main/resources/templates/index.html [hunk 7]
```diff
@@ -1,10 +1,16 @@
 <!DOCTYPE html>
```
Comment (a-kraschitzer) [inline src/main/resources/templates/index.html]: This page should contain spaces for:

>  Informationen über das Hotel wie z.B. Kontakt, Anreise, Aktionen, Wissenswertes, ...

Those spaces can be filled with Lorem Ipsum for now

File: src/main/resources/templates/fragments/header.html [hunk 8]
```diff
@@ -6,13 +6,14 @@
 </head>
 <body>
 <center>
-    <h1>Software Endearing Seminar</h1><br/>
-    <h2>Hotel management site</h2><br/><br/>
     <div id="navcontainer" th:fragment="menu">
+        <h1>Software Endearing Seminar</h1><br/>
```
Comment (a-kraschitzer) [inline src/main/resources/templates/fragments/header.html]: It's "Seminar for Software Engineering" or "Software Engineering Seminar"

Comment (a-kraschitzer) [reply src/main/resources/templates/fragments/header.html]: There should not be any `<h1>` headlines in the header, the header should be a title bar with navigation options and small information, like username.

File: src/main/resources/templates/staff/index.html [hunk 9]
```diff
@@ -6,6 +6,9 @@
     <meta charset="utf-8" />
 </head>
 <body>
+<!--<h1>Hello Spring Security</h1>-->
+<div th:insert="fragments/header :: menu"></div>
+<p>Welcome to the STAFF space!</p>
 <div th:fragment="logout" class="logout" sec:authorize="isAuthenticated()">
```
Comment (a-kraschitzer) [inline src/main/resources/templates/staff/index.html]: move logout button into the header

File: src/main/resources/templates/fragments/header.html (hunk 3 above)
Comment (a-kraschitzer) [reply src/main/resources/templates/fragments/header.html]: hier fehlt noch immer der Link zu staff/roomDirectory

File: src/main/resources/templates/index.html [hunk 10]
```diff
@@ -21,10 +21,12 @@
     </div>
 </div>
 <p>This is an unsecured page, but you can access the secured pages after authenticating.</p>
+<!--
 <ul>
     <li>Go to the <a href="/customer/index" th:href="@{customer/index}">customerEntity pages</a></li>
     <li>Go to the <a href="/staff/index" th:href="@{staff/index}">staff pages</a></li>
 </ul>
+-->
 </body>
```
Comment (a-kraschitzer) [inline src/main/resources/templates/index.html]: Remove unneeded code

File: src/main/resources/templates/staff/index.html (hunk 4 above)
Comment (a-kraschitzer) [reply src/main/resources/templates/staff/index.html]: remove unneeded code

File: pom.xml (hunk 1 above)
Comment (Laszlo32) [reply pom.xml]: done

File: src/main/resources/templates/index.html (hunk 7 above)
Comment (Laszlo32) [reply src/main/resources/templates/index.html]: done (kurzes lorem..)

File: src/main/resources/templates/index.html (hunk 10 above)
Comment (a-kraschitzer) [reply src/main/resources/templates/index.html]: Please implement following draft: [GUI Draft Image](https://nextcloud.kraschitzer.net/index.php/s/kqSa464fGF5CDUa)

File: src/main/resources/templates/fragments/header.html (hunk 2 above)
Comment (Laszlo32) [reply src/main/resources/templates/fragments/header.html]: done

**Pull request merged**